src/sql/sqlops.proposed.d.ts

```diff
@@ -1855,15 +1855,6 @@ declare module 'sqlops' {
 			 */
 			readonly kernelSpec: IKernelSpec;
 
-			/**
-			 * Save the underlying file.
-			 *
-			 * @return A promise that will resolve to true when the file
-			 * has been saved. If the file was not dirty or the save failed,
-			 * will return false.
-			 */
-			save(): Thenable<boolean>;
```

Comment:
I don't think we should remove this. TextDocuments support this functionality, we should do the same for notebooks. Instead of removing, can we update the save implementation in the notebook.component.ts to call through to the input.save instead? Or update src/sql/workbench/api/node/mainThreadNotebookDocumentsAndEditors.ts line 78 to handle it?